pkg/cmd/server/server.go

```diff
@@ -96,7 +98,14 @@ func NewCommand() *cobra.Command {
 			}
 			namespace := getServerNamespace(namespaceFlag)
 
-			s, err := newServer(namespace, fmt.Sprintf("%s-%s", c.Parent().Name(), c.Name()), pluginDir, logger)
+			metricAddressFlag := c.Flag("metric-address")
```

Comment:
Please add a variable at the top of this function (where `logLevelFlag` and `pluginDir` are) and set it to the default address, and then please bind it similar to how we bind `log-level` and `plugin-dir`). Also please call the flag `metrics-address`.